src/shared/pager.c

```diff
@@ -161,8 +161,8 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        stdout = safe_fclose(stdout);
-        stderr = safe_fclose(stderr);
+        safe_fclose(stdout);
+        safe_fclose(stderr);
```

Comment:
I really don't like this patch I must say... stdout/stderr now point to an invalidated object. I'd much rather have this point to NULL than to invalid memory...